platform/features/layout/src/FixedController.js

```diff
@@ -492,42 +505,47 @@ define(
         };
 
         /**
-         * Check if the element is currently selected, or (if no
-         * argument is supplied) get the currently selected element.
-         * @returns {boolean} true if selected
+         * Checks if the element should be selected or not.
+         *
+         * @param elementProxy the element to check
+         * @returns {boolean} true if the element should be selected.
          */
-        FixedController.prototype.selected = function (element) {
-            var selection = this.selection;
-            return selection && ((arguments.length > 0) ?
-                    selection.selected(element) : selection.get());
+        FixedController.prototype.selected = function (elementProxy) {
```

Comment:
Given the changes to functionality (mentioned in JSDoc comment) I'd change the method name to `shouldSelect`